ball_finder.py

```diff
@@ -0,0 +1,81 @@
+import cv2
+import math
+import numpy as np
+
+
+class BallFinder:
+    def __init__(self, detector):
+        self.detector = detector
+        self.generator = detector.get_generator()
+
+        frame, _ = self.generator.generate()
+        self.SCREEN_HEIGHT, self.SCREEN_WIDTH = frame.shape[:2]
+        self.FOCAL_LENGTH_PIXELS = 657
+
+        # Depth csv of empty floor
+        self.floor_frame = np.loadtxt("FLOOR.csv", dtype=np.float32, delimiter=",")
+        self.floor_frame[self.floor_frame > 4] = 0
```

Comment:
Again, I think you'd be better off setting ignored pixels to infinity distance.

Reply:
Could you clarify how that would affect the values we get? As it stands, both 0 and infinity are outside of the bounds.

Reply:
The reason we didn't change it to infinity is because we wanted to set a min distance as well (the beginning of the robot) I think were confused as to how setting it to infinity would make it simpler.